admin/kernel-syncdata-service/src/main/java/io/mosip/kernel/syncdata/service/helper/ClientSettingsHelper.java

```diff
@@ -49,6 +49,9 @@ public class ClientSettingsHelper {
 	@Autowired
 	private ClientCryptoManagerService clientCryptoManagerService;
 
+	@Value("${mosip.syncdata.regclient.support114:true}")
```

Comment:
Default value lets keep this false, so it avoids sending these extra data for 115 users, which is most of the countries

Reply:
set it to false by default